src/Signature.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PharIo\GnuPG;
+
+/**
+ * This class could be prevent sigatures form being serialized.
+ *
+ * @see https://github.com/paragonie/hidden-string/blob/master/src/HiddenString.php inspired by.
```

Comment:
I don't see this for `Signature`. There is nothing secret about it.

It would make sense though to do this for the `SecretKeyData` class.